static/scss/product-details.scss

```diff
@@ -1,5 +1,5 @@
 // sass-lint:disable mixins-before-declarations
-.container {
+.container:not(.auth-page) {
```

Comment:
Why would a css file called product-details apply to an auth page to begin with?

Reply:
@Wassaf-Shahzad Agreed w/ Peter. Instead of de-selecting the auth page container elements here, we need to make this selector specific to the product page. All other `.container` elements should not be selected by this. I don't want to create a new page in the future, use a `.container` and have these non-standard styles applied to it. We need to add a class to one of the top-level elements in the product page (`.product-page` or something like that), then change this selector to `.product-page .container`

Reply:
@gsidebo As requested I have added a parent class to the top level element of product-page to stop style leaking into other elements.